builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -455,6 +455,168 @@ func TestAccVSphereVirtualMachine_createWithCdrom(t *testing.T) {
 	})
 }
 
+func TestAccVSphereVirtualMachine_updateMemory(t *testing.T) {
+	var vm virtualMachine
+	var locationOpt string
+	var datastoreOpt string
+
+	if v := os.Getenv("VSPHERE_DATACENTER"); v != "" {
```

Comment:
provider_test.go only checks for VSPHERE_USER, VSPHERE_PASSWORD and VSPHERE_SERVER, but not VSPHERE_DATACENTER.